app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -830,6 +853,37 @@ class AdvancedFilter extends React.Component {
     );
   };
 
+  /**
+   * Renders multi-select type line "statement"
+   * @param {Object} filter - Filter currently selected
+   * @param {Number} index - Filter index
+   * @param {Array} value - Current values selected
+   */
+  renderMultiStatement = (filter, index, value) => {
+    // Tooltip for Multi-select type
+    let tooltip = 'Leaving this field blank will not filter out any monitorees.';
+
+    return (
+      <React.Fragment>
+        <div className="d-flex justify-content-between py-0 my-0">
+          <Select
+            closeMenuOnSelect={false}
+            isMulti
+            value={value}
+            options={filter.options}
```

Comment:
When I go to edit these after saving/applying, the options are not there

![Screen Shot 2021-08-10 at 1 20 38 PM](https://user-images.githubusercontent.com/9956561/128905644-c699bec9-f8f2-4bc5-8716-9a8b72183d85.png)

Reply:
will look into this. Thanks, Tim!